ddsrouter_test/compose/test_cases/stress/repeater_mixed_data_sizes/compose.yml

```diff
@@ -0,0 +1,398 @@
+# Test description:
+#   This test stresses a router with a repeater participant by connecting to it 14 different edge routers
+#   and publishing messages of different sizes (1B, 12B, 123B, 1.23KB, 12.3KB, 123KB, 1.23MB) in the same topic.
+#
+# Test architecture:
```

Comment:
Improve the schema with detailed information about the networks, sizes sent by each publisher...